packages/system/data_stream/auth/_dev/test/pipeline/test-timestamp.log-config.yml

```
@@ -1,4 +1,5 @@
 fields:
-  event.timezone: "+0000"
+  event:
+    timezone: "+0000"
```

```suggestion
  event:
    timezone: "+0000"
```

@jsoriano If I am not wrong this also needs to be quoted, right?

PR with similar changes: https://github.com/elastic/integrations/pull/8174

---

It seems to only fail for `dynamic_fields`

---

We have reverted the change in `elastic-package` that was requiring all these changes in test config files. So you can also revert all these changes, they should not be needed anymore.